action.yml

```diff
@@ -58,12 +67,28 @@ runs:
         max_attempts: ${{ inputs.max_attempts }}
         retry_wait_seconds: ${{ inputs.retry_wait_seconds }}
         shell: bash
-        command: stepzen login ${{ inputs.domain }} --account ${{ inputs.account }} --adminkey ${{ inputs.adminkey }}
+        command: |
+          if [ "${{ inputs.account }}" != "" ]; then
```

Comment:
Primary mechanism should be using the instance id, so if the user provides the instance id and account, login is through the instance id.